azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/utilities/BottomCellActionViewHolder.kt

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.utilities
+
+import android.view.View
+import android.widget.ImageView
+import androidx.core.content.ContextCompat
+import androidx.core.view.AccessibilityDelegateCompat
+import androidx.core.view.ViewCompat
+import androidx.core.view.accessibility.AccessibilityNodeInfoCompat
+import com.azure.android.communication.ui.R
+import com.azure.android.communication.ui.calling.utilities.BottomCellItem
+import com.azure.android.communication.ui.calling.utilities.BottomCellItemType
+import com.azure.android.communication.ui.calling.utilities.BottomCellViewHolder
+import com.microsoft.fluentui.persona.AvatarView
+
+internal class BottomCellActionViewHolder(itemView: View) : BottomCellViewHolder(itemView) {
+    private val imageView: ImageView = itemView.findViewById(R.id.cell_icon)
+    private val avatarView: AvatarView =
+        itemView.findViewById(R.id.azure_communication_ui_participant_list_avatar)
+    private val accessoryImageView: ImageView = itemView.findViewById(R.id.cell_check_mark)
+
+    override fun setCellData(bottomCellItem: BottomCellItem) {
+        super.setCellData(bottomCellItem)
+        if (bottomCellItem.itemType == BottomCellItemType.BottomMenuTitle) {
+            // Exit with early return because setting the title is the only thing we need to do
+            return
+        }
+        // force bitmap update be setting resource to 0
+        avatarView.setImageResource(0)
+        if (bottomCellItem.icon == null) {
+            ViewCompat.setAccessibilityDelegate(
+                itemView,
+                object : AccessibilityDelegateCompat() {
+                    override fun onInitializeAccessibilityNodeInfo(
+                        host: View,
+                        info: AccessibilityNodeInfoCompat,
+                    ) {
+                        super.onInitializeAccessibilityNodeInfo(host, info)
+                        info.removeAction(AccessibilityNodeInfoCompat.AccessibilityActionCompat.ACTION_CLICK)
+                        info.isClickable = false
+                    }
+                }
+            )
+            imageView.visibility = View.GONE
+            avatarView.visibility = View.VISIBLE
+            avatarView.name = bottomCellItem.title ?: ""
+            bottomCellItem.participantViewData?.let { participantViewData ->
+                participantViewData.avatarBitmap?.let {
+                    avatarView.avatarImageBitmap = it
+                    avatarView.adjustViewBounds = true
+                    avatarView.scaleType = participantViewData.scaleType
+                }
+            }
+        } else {
+            imageView.setImageDrawable(bottomCellItem.icon)
+            avatarView.visibility = View.GONE
+        }
+        if (bottomCellItem.accessoryImage != null) {
```

Comment:
more concise and idiomatic to use ?.let { } instead

Reply:
Included in 0beaa7d